.github/ISSUE_TEMPLATE/2_improvement.md

```diff
@@ -0,0 +1,84 @@
+---
+name: Improvement proposal
+about: Improvements to the contracts, the website, the copy... Anything that is working well but could be enhanced.
```

Comment:
What is `copy...`?

Reply:
do you mean the tests?, what is the copy 🧐

Reply:
Copy as in copywriting, so changes in the text on the website for instance.

Reply:
Which website exactly? Do we have one for the migaloo core repo?

Reply:
Not a specific one, can be the porject or the app website, it's more about general improvements across the project.